client/src/pages/ExistingProgramme/index.js

```diff
@@ -30,15 +32,22 @@ const ExistingProgramme = ({ id }) => {
                 </Text>
             </Flex>
 
-            <Flex align="center" justify="center" marginTop="50px">
+            <Flex align="center" justify="center" direction="column">
                 <Text textAlign="center" fontSize="md" color="#000000" as="i">
                     Requirements
                 </Text>
-                <Box as="span" ml="2" color="gray.600" bg="gray" />
+                <Box bg="#e2e2e2" width="75%" textAlign="center">
+                    <IconButton width="90%" icon="add" bg="#616161" color="white" onClick={() => setCreateNewRequirements(true)} />
```

Comment:
Same as above, let's add some more padding so the button doesn't touch the top of the box

Reply:
done